### DISCUSSION TITLE
Is it possible to export the history of all hardwares?

### DISCUSSION THREAD

**Discussion by chazkame** [snipe/snipe-it#14442] [Q&A]
Hello,

Is it possible to export all the hardwares' history table? (Check-in/Check-out, maintenance, etc.)

**Comment by ShaunNeighbourSwale** [comment]
For maintenance, you gate them here - https://snipe-it.readme.io/reference/maintenances

For all other history, you can go here - https://snipe-it.readme.io/reference/reportsactivity
